Sources/MockoloFramework/Models/ParamModel.swift

```diff
@@ -24,39 +24,39 @@ final class ParamModel: Model {
     var type: Type
     let label: String
     let isGeneric: Bool
-    let isInitializer: Bool
+    let inInit: Bool
```

Comment:
What's the purpose of this change?

Reply:
this description is more accurate than is*